assets/app/views/browse/build-config.html

```diff
@@ -28,6 +28,9 @@
                 ><i class="fa fa-play fa-fw" aria-hidden="true"></i> Start Build</a>
               </li>
               <li>
+                <a ng-href="{{buildConfig | editResourceURL}}" role="button"><i class="fa fa-pencil fa-fw" aria-hidden="true"></i> Edit </a>
```

Comment:
Edit YAML seems ok when you also have Edit, but when you only have Edit
YAML would you understand thats how you edit the object?

On Tue, Jan 5, 2016 at 8:42 AM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/browse/build-config.html
> <https://github.com/openshift/origin/pull/6528#discussion_r48845170>:
>
> > @@ -28,6 +28,9 @@
> >                  ><i class="fa fa-play fa-fw" aria-hidden="true"></i> Start Build</a>
> >                </li>
> >                <li>
> > +                <a ng-href="{{buildConfig | editResourceURL}}" role="button"><i class="fa fa-pencil fa-fw" aria-hidden="true"></i> Edit </a>
>
> If we change the "Edit (Raw)" to "Edit YAML", the difference might be
> clearer. @jwforres <https://github.com/jwforres> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6528/files#r48845170>.
>